server.js

```diff
@@ -65,12 +65,52 @@ app.post('/api/v1/images', (req, res) => {
         })
     }
   }
-  const {id, url, title, color, artist, type } = addedImage
-  app.locals.images.push({ id, url, title, color, artist, type })
-  res.status(201).json({ id, url, title, color, artist, type })
+  const {id, url, title, color, artist, type, quantity, price } = addedImage
+  app.locals.images.push({ id, url, title, color, artist, type, quantity, price })
+  res.status(201).json({ id, url, title, color, artist, type, quantity, price })
   console.log(req, res)
 })
 
+// GET cart items
+app.get('/api/v1/cart', (req, res) => {
+  const cartItems = app.locals.cart
+  // console.log('RequestPARAMS>>>', req.params)
+  // console.log('Response>>>', res)
+  res.json(cartItems)
+})
+
+// POST item to cart
+app.post('/api/v1/cart', (req, res) => {
```

Comment:
Nice, glad we got this part figured out. Nice to know that we can pull apart `request.body` like this if we want to manipulate its properties and then put it back together